src/main/java/com/iemr/common/controller/users/IEMRAdminController.java

```diff
@@ -933,9 +938,14 @@ public String forceLogout(@RequestBody ForceLogoutRequestModel request, HttpServ
 	    try {
 	        // Perform the force logout logic
 	        iemrAdminUserServiceImpl.forceLogout(request);
-
+	        String token = null;
+	        token = getJwtTokenFromCookies(httpRequest);
+	        if(null == token) {
+	        	token = httpRequest.getHeader(Constants.JWT_TOKEN);
+	        }
+	        TokenBlacklist.blacklistToken(token,BLACK_LIST_EXPIRATION_TIME);
 	        // Extract and invalidate JWT token cookie dynamically from the request
-	        invalidateJwtCookie(httpRequest, response);
+	       // invalidateJwtCookie(httpRequest, response);
 
```

Comment:
If this method is not going to be used, let's remove it from code.
Setting cookie to null with Set-cookie header is still valid though.